modules/compose/compose_builder_test.go

```diff
@@ -158,19 +154,15 @@ func writeTemplateWithSrvType(t *testing.T, templateFile string, srvType string,
 	values["ServiceType"] = srvType
 
 	output, err := os.Create(composeFile)
-	if err != nil {
-		t.Fatalf("creating output file: %s", err)
-	}
+	require.NoErrorf(t, err, "creating output file")
 	defer output.Close()
```

Comment:
bug: this can loose a write error. Not related to this PR, but lets fix while here
```go
defer func() {
    require.NoError(t, outpout.Close())
}()
```